### ISSUE TITLE
[Web] Zooming in/out into a picture on a touchscreen laptop is too sensitive.

### ISSUE THREAD

**Issue by yurividal** [immich-app/immich#6896]
### The bug

I have a touchscreen laptop, and when I am browsing immich and i try to zoom in to a picture, it is way too sensitive and zooms a LOT with minimal finger movement. it feels unatural and inconsistent with the way other websites work. 

### The OS that Immich Server is running on

Ubuntu 22.10 - Docker

### Version of Immich Server

1.94.1

### Version of Immich Mobile App

1.94.1

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Irrelevant to this bug
```

### Your .env content

```Shell
Irrelevant to this bug
```

### Reproduction steps

```bash
Login to immmich on a browser. 
Open a picture
try to zoom in with the touchscreen
```

**Comment by iqapps** [comment]
I have the Samsung Galaxy Tab 8, and using web access for immich, zoom is way to sensitive here too.

**Comment by Pascal-So** [comment]
I can also confirm this on version 1.103.1, and I believe this probably has to be fixed in the zoom-image library that immich uses for this functionality. See the related issue there: https://github.com/willnguyen1312/zoom-image/issues/283

**Comment by Pascal-So** [comment]
The corresponding PR https://github.com/willnguyen1312/zoom-image/pull/284 is now merged, so this should be fixed with the next dependency update!

**Comment by Pascal-So** [comment]
Just checked, it's fixed in 1.106, so I'd say this issue can be closed :)
